### PR TITLE
feat(OOPIF): debug message in case of iFrame moved to another process

### PR DESCRIPTION
**Pull request by sadym-chromium**
Debug message clarifying the problem like this: [`Issue 1060080: Event Network.loadingFinished is not sent when a frame is loaded from another domain`](https://bugs.chromium.org/p/chromium/issues/detail?id=1060080).
It can help users to identify problem with OOPIF easier without digging into the CDP protocol implementation like in the bug `1060080` mentioned above.

To reproduce:
1. Run `DEBUG="puppeteer:frame" NODE_PATH=../ node examples/oopif.js`.
1. Verify the output contains the debug message:
```
puppeteer:frame The frame '...' moved to another session. Out-of-proccess
iframes (OOPIF) are not supported by Puppeteer yet.
https://github.com/puppeteer/puppeteer/issues/2548
```

### CODE REVIEW

File: src/common/FrameManager.ts [hunk 1]
```diff
@@ -213,6 +219,20 @@ export class FrameManager extends EventEmitter {
...
   }
 
+  private async _onFrameMoved(event: Protocol.Target.AttachedToTargetEvent) {
+    if (event.targetInfo.type !== 'iframe') {
+      return;
+    }
+
+    // TODO(sadym): Remove debug message oncle the proper OOPIF is implemented:
+    // https://github.com/puppeteer/puppeteer/issues/2548
+    debugFrames(
```
Comment (mathiasbynens) [inline src/common/FrameManager.ts]: Since we only need `debugFrames` once, let’s move its definition here.

File: src/common/FrameManager.ts [hunk 2]
```diff
@@ -213,6 +219,20 @@ export class FrameManager extends EventEmitter {
...
+      `The frame '${event.targetInfo.targetId}' moved to another session. ` +
+        `Out of proccess iFrames (OOPIF) are not supported by Puppeteer yet. ` +
```
Comment (mathiasbynens) [inline src/common/FrameManager.ts]:
```suggestion
        `Out-of-process iframes (OOPIF) are not supported by Puppeteer yet. ` +
```

File: src/common/FrameManager.ts [hunk 3]
```diff
@@ -213,6 +219,20 @@ export class FrameManager extends EventEmitter {
...
+        `The feature-request: https://github.com/puppeteer/puppeteer/issues/2548`
```
Comment (mathiasbynens) [inline src/common/FrameManager.ts]:
```suggestion
        `https://github.com/puppeteer/puppeteer/issues/2548`
```

**Review by mathiasbynens — CHANGES_REQUESTED** [review]

File: src/common/FrameManager.ts [hunk 4]
```diff
@@ -213,6 +219,20 @@ export class FrameManager extends EventEmitter {
     return watcher.navigationResponse();
   }
 
+  private async _onFrameMoved(event: Protocol.Target.AttachedToTargetEvent) {
+    if (event.targetInfo.type !== 'iframe') {
+      return;
+    }
+
+    // TODO(sadym): Remove debug message oncle the proper OOPIF is implemented:
```
Comment (mathiasbynens) [inline src/common/FrameManager.ts]:
```suggestion
    // TODO(sadym): Remove debug message once proper OOPIF support is implemented:
```

File: src/common/FrameManager.ts (hunk 1 above)
Comment (sadym-chromium) [reply src/common/FrameManager.ts]: done

File: src/common/FrameManager.ts (hunk 2 above)
Comment (sadym-chromium) [reply src/common/FrameManager.ts]: done

File: src/common/FrameManager.ts (hunk 4 above)
Comment (sadym-chromium) [reply src/common/FrameManager.ts]: done

File: src/common/FrameManager.ts [hunk 5]
```diff
@@ -213,6 +218,20 @@ export class FrameManager extends EventEmitter {
...
+  private async _onFrameMoved(event: Protocol.Target.AttachedToTargetEvent) {
+    if (event.targetInfo.type !== 'iframe') {
+      return;
+    }
+
+    // TODO(sadym): Remove debug message once proper OOPIF support is 
+    // implemented: https://github.com/puppeteer/puppeteer/issues/2548
+    debug('puppeteer:frame')(
+      `The frame '${event.targetInfo.targetId}' moved to another session. ` +
+        `Out-of-proccess iframes (OOPIF) are not supported by Puppeteer yet. ` +
```
Comment (mathiasbynens) [inline src/common/FrameManager.ts]: Hmm, I thought I had fixed this typo already?

```suggestion
        `Out-of-process iframes (OOPIF) are not supported by Puppeteer yet. ` +
```

**Review by mathiasbynens — APPROVED** [review]
LGTM % remaining typo. Please include a minimal example script that triggers this new code path in the description + commit message.

**Pull request merged**